utils/namespaces.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+type Namespaces struct {
+	Namespace map[string]string `json:"namespace"`
+}
+
+func GetNamespaces(bucket, namespace string, client *s3.Client) ([]string, error) {
+
+	byteValue, _, err := ImportS3File(client, bucket, "namespace_costs.json")
```

Comment:
Getting the namespaces this way might be problematic in the future, let's say the namespace_costs job failed for whatever reason, it would then cause lots of unrelated jobs to fail. Try generating the list of namespaces in a different way.

Reply:
Also, I think you can delete this file because if we use dynamic routing we won't need to loop over all the namespaces like we do atm